gcloud/bigtable/test_cluster.py

```diff
@@ -363,3 +383,12 @@ def __init__(self, project, timeout_seconds=None):
         self.project = project
         self.project_name = 'projects/' + self.project
         self.timeout_seconds = timeout_seconds
+
+    def copy(self):
+        from copy import deepcopy
+        return deepcopy(self)
+
+    def __eq__(self, other):
```

Comment:
@tseaver PTAL. I implemented `__eq__` on the mock `_Client` and then got rid of the `__dict__` check and the `Cluster._client` patch before `assertEqual` in `test_copy` above.